instance/serializers/instance.py

```diff
@@ -110,3 +110,15 @@ class InstanceReferenceDetailedSerializer(InstanceReferenceBasicSerializer):
     more detail.
     """
     summary_only = False
+
+
+class InstanceLogSerializer(serializers.ModelSerializer):
+    """
+    Provide the log entries for an Instance
+    """
+    log_entries = LogEntrySerializer(many=True, read_only=True)
+    log_error_entries = LogEntrySerializer(many=True, read_only=True)
+
+    class Meta:
+        model = InstanceReference
+        fields = ('log_entries', 'log_error_entries',)
```

Comment:
`Instance` does not have a `log_error_entries` field, so that isn't doing anything - please remove it from this serializer.